### PR TITLE
Adds a wrapped version of the grpc-java library

### PR DESCRIPTION
**Pull request by eamansour**
For https://github.com/galasa-dev/projectmanagement/issues/1568

Wraps the [grpc-java](https://github.com/grpc/grpc-java) library dependencies into an OSGi bundle to allow for Java gRPC clients to be generated from .proto files. The following dependencies are wrapped:

`io.grpc:grpc-netty-shaded:1.60.0'` - https://mvnrepository.com/artifact/io.grpc/grpc-netty-shaded/1.60.0
`'io.grpc:grpc-protobuf:1.60.0'`- https://mvnrepository.com/artifact/io.grpc/grpc-protobuf/1.60.0
`'io.grpc:grpc-stub:1.60.0'` - https://mvnrepository.com/artifact/io.grpc/grpc-stub/1.60.0

### CODE REVIEW

**Comment by galasa-team** [comment]
Build successful

**Review by techcobweb — APPROVED** [review]

File: io.grpc.java/pom.xml [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+		</dependency>
+		<dependency>
+			<groupId>com.google.errorprone</groupId>
+			<artifactId>error_prone_annotations</artifactId>
+			<version>2.20.0</version>
+		</dependency>
+		<dependency>
+			<groupId>com.google.guava</groupId>
+			<artifactId>guava</artifactId>
+			<version>32.0.1-android</version>
```
Comment (techcobweb) [inline io.grpc.java/pom.xml]: surprised to see android mentioned

File: io.grpc.java/pom.xml [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+		</dependency>
+		<dependency>
+			<groupId>com.google.guava</groupId>
+			<artifactId>failureaccess</artifactId>
+			<version>1.0.1</version>
+		</dependency>
+		<dependency>
+			<groupId>com.google.guava</groupId>
+			<artifactId>listenablefuture</artifactId>
+			<version>9999.0-empty-to-avoid-conflict-with-guava</version>
```
Comment (techcobweb) [inline io.grpc.java/pom.xml:98]: What's that all about ? A comment would be good ?

File: io.grpc.java/pom.xml [hunk 3]
```diff
@@ -0,0 +1,161 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
```
Comment (techcobweb) [inline io.grpc.java/pom.xml:1]: Why are we still using maven for this ?

**Review by techcobweb — APPROVED** [review]
Approved, but some comments might be good to answer.

File: io.grpc.java/pom.xml (hunk 1 above)
Comment (eamansour) [reply io.grpc.java/pom.xml]: The `grpc-netty-shaded`, `grpc-protobuf`, and `grpc-stub` packages depend on this specific version of guava - see:
https://mvnrepository.com/artifact/io.grpc/grpc-netty-shaded/1.60.0
https://mvnrepository.com/artifact/io.grpc/grpc-protobuf/1.60.0
https://mvnrepository.com/artifact/io.grpc/grpc-stub/1.60.0

Edit: interestingly, after switching to `32.0.1-jre`, the changes still work - so I've changed the above to use that version instead.

File: io.grpc.java/pom.xml (hunk 2 above)
Comment (eamansour) [reply io.grpc.java/pom.xml:98]: Happy to add a comment - this is the published version of one of guava's dependencies - see https://mvnrepository.com/artifact/com.google.guava/guava/32.0.1-android

File: io.grpc.java/pom.xml (hunk 3 above)
Comment (eamansour) [reply io.grpc.java/pom.xml:1]: All of the bundles in wrapping use maven - what do you recommend they should use?

**Comment by galasa-team** [comment]
Build successful

**Pull request merged**